client/app/email-domain/email/filter/email-domain-email-filter.html

```diff
@@ -12,71 +12,31 @@ <h2>
-                            <td class="text-center">
-                                <span class="label"
-                                      data-ng-class="{'label-success': filter.active, 'label-danger': !filter.active}"
-                                      data-ng-bind="tr('email_tab_filters_status_' + filter.active)"></span>
-                            </td>
-                            <td class="text-center">
-                                <button class="btn btn-icon" type="button"
-                                        data-linkedpopover="email-domain/email/filter/email-domain-email-filter.popover.html"
-                                        data-linkedpopover-placement="left"
-                                        data-linkedpopover-remote="true"
-                                        data-linkedpopover-single="true"
-                                        data-linkedpopover-title="{{i18n.common_actions}}">
-                                    <span class="oui-icon oui-icon-gear_line oui-icon_small" aria-hidden="true"></span>
-                                </button>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-        </div>
-
-        <div class="clearfix"
-             data-ng-if="ctrlEmailDomainEmailFilter.filters">
-            <div data-pagination-front
-                 data-items="ctrlEmailDomainEmailFilter.filters"
-                 data-paginated-items="ctrlEmailDomainEmailFilter.filtersDetails"
-                 data-current-page="currentPageFilters"
-                 data-nb-pages="nbPages"
-                 data-items-per-page="itemsPerPage"
-                 data-transform-item="ctrlEmailDomainEmailFilter.transformItem(item)"
-                 data-on-transform-item-done="ctrlEmailDomainEmailFilter.onTransformItemDone()"
-                 data-page-placeholder="{{tr('pagination_page', [currentPageFilters, nbPages])}}"
-                 data-item-per-page-placeholder="{{tr('pagination_display')}}"
-                 data-ng-show="!ctrlEmailDomainEmailFilter.loading.filters && ctrlEmailDomainEmailFilter.filters.length > 0">
-            </div>
-        </div>
+        <oui-datagrid data-rows="ctrlEmailDomainEmailFilter.filters"
+                      data-row-loader="ctrlEmailDomainEmailFilter.transformItem($row)">
+            <oui-column data-title="tr('email_tab_table_header_filter_name')" data-property="name" data-sortable="asc"></oui-column>
+            <oui-column data-title="tr('email_tab_table_header_filter_priority')" data-property="priority" data-sortable></oui-column>
+            <oui-column data-title="tr('emails_common_status')">
+                <span class="label"
+                      data-ng-class="{'label-success': $row.active, 'label-danger': !$row.active}"
+                      data-ng-bind="tr('email_tab_filters_status_' + $row.active)"></span>
```

Comment:
data-i18n-static would be a better option and single binded

Reply:
same